### PR TITLE
FR-1406: Re-Adding Example data loader notebook

### PR DESCRIPTION
**Pull request by CK-Gea**
# Pull Request Checklist

- [x] Changes follow the [style guide](https://github.com/finbourne/sample-notebooks/blob/master/docs/FINBOURNE%20notebook%20style%20guide.ipynb)
- [x] Tests pass
- [ ] Raised the PR against the `develop` branch

# Description of the PR

Re adding previously removed example data loader notebook.

### CODE REVIEW

File: examples/example-data/Example Data Load.ipynb [hunk 1]
```diff
@@ -0,0 +1,718 @@
+{
```
Comment (rizwansaeed) [inline examples/example-data/Example Data Load.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #25.        valuation_schedule=models.ValuationSchedule(effective_at=pd.to_datetime('2021-05-17', utc=True)),
</pre><p> </p><p><code>pd.to_datetime('2021-05-17', utc=True)</code>  needs to be <code>pd.to_datetime('2021-05-17', utc=True).isoformat()</code> (its always meant to be a string but there is now validation on this)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/619/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

**Comment by rizwansaeed** [comment]
can you raise against the `develop` branch

File: examples/example-data/Example Data Load.ipynb (hunk 1 above)
Comment (CK-Gea) [reply examples/example-data/Example Data Load.ipynb:1]: <p>done</p>

**Pull request merged**